src/features/Main/ActiveTabUpdater/index.jsx

```diff
@@ -0,0 +1,16 @@
+import { useEffect } from 'react';
+import { useDispatch } from 'react-redux';
+import { updateActiveTab } from 'features/Main/data/slice';
+
+const ActiveTabUpdater = ({ children }) => {
```

Comment:
@01001110J Following @AuraAlba Comment I seeing the same issue.

![2024-04-0519-14-21-ezgif com-video-to-gif-converter](https://github.com/Pearson-Advance/frontend-app-institution-portal/assets/30726391/6b08a188-afaf-4b2b-a5f5-8edf43141e56)

The active tab needs to be validated two times.